config/zephyr/Kconfig

```diff
@@ -479,6 +479,18 @@ config CHIP_FACTORY_RESET_ERASE_NVS
 	  configuration is supposed to be cleared on a factory reset, including
 	  device-specific entries.
 
+config CHIP_FACTORY_RESET_ERASE_ZMS
+	bool "Erase NVS flash pages on factory reset"
```

Comment:
Should prompt and help be exactly the same as for `CHIP_FACTORY_RESET_ERASE_NVS` ?

Reply:
right, it shouldn't, thanks!